Some updates to the ksql workshop based on my trial run testing 5.0.x

rodesai opened the pull request:
- Include OSX in the note on the network naming
- Add some details on how to access kibana

asnir commented:
@miguno

miguno commented:
@asnir:  This PR introduces quite a few changes that are not related to each other (e.g. the introduction of some docker functionality vs. more meaningful variable names vs. better CLI parsing).  Some of these changes are easier/quicker to accept and review than others, so if it works for you I'd suggest to split the PR into separate ones (which can then also have more descriptive PR titles rather than a generic "code refactoring").

miguno commented on kafka-clients/consumer/pom.xml:
```diff
@@ -35,7 +35,7 @@
 
     <properties>
         <java.version>1.7</java.version>
-        <kafka.version>0.9.0.1-cp1</kafka.version>
```
For now we want to keep the version at `0.9.0.1-cp1`.

miguno commented on kafka-clients/consumer/pom.xml:
```diff
@@ -88,6 +88,21 @@
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
```
Why the shading?

miguno commented:
Closed because of inactivity. Feel free to reopen if needed (and see review comments above).

rodesai committed:
Some updates to the ksql workshop based on my trial run testing 5.0.x

- Include OSX in the note on the network naming
- Add some details on how to access kibana

rodesai committed:
Fix typo

ybyzek commented:
@rmoff @rodesai can y'all provide an update to this PR to refresh it or close the PR?
